src/curvemap.h

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 1997-2024 The CSE Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file.
+
+// curvemap.h -- include file for curve and performance map
+
+#ifndef _CURVEMAP_H
+#define _CURVEMAP_H
+
+enum class PMSPEED { MIN, RATED, MAX };
+
+class PMACCESS
```

Comment:
This class is very specific to the RSYS application. Is that intended?

Reply:
Will generalize when/if needed

Reply:
I think we rename for now.